### PR TITLE
content(cms): create Tag "ai"

### PR DESCRIPTION
**Pull request by emilykateemilykate**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by ttasovac** [comment]
@emilykateemilykate you shouldn't be creating new tags yourself, unless you have confirmed that with @VickyGarnett first (if you have —  i apologize for the confusion).  also, the tag description should define the tag —  in this case, what artificial intelligence is rather than saying "resources that have to do with artificial intelligence" —  because that doesn't tell the user much...

**Comment by emilykateemilykate** [comment]
Understood- thanks so much!

**Comment by VickyGarnett** [comment]
Hi @ttasovac  - I've had a chat with Emily, and I see value in adding this tag, as we already have a few AI-based resources on Campus.  I've updated the definition. - V

File: content/tags/ai.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name: Artificial Intelligence
+description: Artificial Intelligence (AI) is the ability for computer systems to perform tasks that would normally require human intelligence
```
Comment (ttasovac) [inline content/tags/ai.yml]: @VickyGarnett, how about: Artificial Intelligence (AI) is a branch of computer science aimed at developing computer systems that can solve complex problems by simulating human cognitive abilities

Comment (VickyGarnett) [reply content/tags/ai.yml]: Even better, thanks @ttasovac

**Pull request merged**